tests/browser/spa/api.browser.js

```diff
@@ -6,7 +6,7 @@
 let originalSetTimeout = global.setTimeout
 
 const jil = require('jil')
-const {getInfo} = require('../../../packages/browser-agent-core/common/config/config')
+const {getInfo} = require('@newrelic/browser-agent-core/src/common/config/config')
```

Comment:
Here is a third way that package code is being imported.  We should pick one

Reply:
We should use the package name import approach to make life simpler. Let me go back through the tests code and make sure they are all using the correct imports.